src/themes/teams/components/Segment/segmentStyles.ts

```diff
@@ -1,10 +1,13 @@
+import * as _ from 'lodash'
+
 import { SegmentProps } from '../../../../components/Segment/Segment'
 import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
 import { SegmentVariables } from './segmentVariables'
 
 const segmentStyles: ComponentSlotStylesInput<SegmentProps, SegmentVariables> = {
   root: ({ props: p, variables: v }): ICSSInJSStyle => {
-    const color = p.color || v.color
+    const color = _.get(v.colors, p.color, v.color)
```

Comment:
this logic appears in lots of places already - we should generalize it and consume from the single place

Reply:
`getColmputedColor` -> `getComputedColor`?

yes, there is no much gain here, agree - lets leave as it is and see how it could be better generalized (if it should be). Thank you!